### PR TITLE
feat(VBC-115): setup guest wishlist composable

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
Setup guest wishlist composable

## Description
Basic configuration for guest wishlist composable.

### CODE REVIEW

File: packages/composables/src/types/index.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { IntegrationContext } from '@vue-storefront/core';
 import { ContextualizedEndpoints } from '@vue-storefront/bigcommerce-api';
 import { AxiosInstance } from 'axios';
 
+export * from './useCategory';
```
Comment (dhargitai) [inline packages/composables/src/types/index.ts:5]: Why did you add this here?

File: packages/composables/src/useGuestWishlist/index.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import { Ref, computed } from '@vue/composition-api';
+import { sharedRef, UseReviewErrors/* , Logger, generateContext */ } from '@vue-storefront/core';
```
Comment (dhargitai) [inline packages/composables/src/useGuestWishlist/index.ts:2]: Could you delete these unused/commented parts? (in line 2 and 4)

**Review by dhargitai — COMMENTED** [review]
Looks good, I only left 2 minor comments. :)

File: packages/composables/src/types/index.ts (hunk 1 above)
Comment (WojtekTheWebDev) [reply packages/composables/src/types/index.ts]: I added it by mistake here, thanks

**Review by dhargitai — APPROVED** [review]

File: packages/composables/src/useGuestWishlist/index.ts (hunk 2 above)
Comment (WojtekTheWebDev) [reply packages/composables/src/useGuestWishlist/index.ts]: Done :)
